contact_detection/src/contact_detection.cpp

```diff
@@ -0,0 +1,25 @@
+#include "contact_detection/contact_detection.h"
+
+contact_detection::contact_detection(ros::NodeHandle nh) {
+	nh_ = nh;
+
+	// Load rosparams from parameter server
+	nh.param<double>("ekf_estimator/update_rate", update_rate_, 200); // add a param for your package instead of using the estimator one
```

Comment:
Add config parameter contact_detection/update_rate to config/project_params.yaml and reference here